static/partials/settings.html

```diff
@@ -283,6 +283,31 @@
             </div>
           </div>
 
+          <div ng-show="withTurnCustomizable">
```

Comment:
This is yet another name for this feature, i see withTurnCustomizable, preferClientSideTurnSettings, TurnClientSideSetable. Three different names for the same feature. I do not like any of these names very much. Make this consistent and use a better name, like ownTurn or if you must clientSideTurn and use it consistently everwhere.